deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -342,6 +342,14 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resourceNames:
+          - k8s
+          resources:
+          - prometheuses/api
+          verbs:
+          - get
```

Comment:
You mentioned offline we might not need the RBAC anymore. Can we remove it?

Reply:
We still need it, I required it during aws testing